### PR TITLE
Make work on Rocky9 platform

### PR DESCRIPTION
**Pull request by pwalczysko**
This is passing on https://github.com/pwalczysko/ansible-role-nfs-share/actions/runs/6559321978

cc @khaledk2 @jburel @sbesson

### CODE REVIEW

**Review by sbesson — COMMENTED** [review]
Exact same feedback as https://github.com/ome/ansible-role-nfs-mount/pull/10#pullrequestreview-1684699459 i.e. :+1: for the changes. The enabling of the GitHub Actions and the `destroy.yml` phase might need to be investigated

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -9,10 +9,15 @@ lint: |
     flake8
 platforms:
   - name: nfs-share
-    image: centos/systemd
+    image: eniocarboni/docker-rockylinux-systemd:9
```
Comment (joshmoore) [inline molecule/default/molecule.yml:12]: Just as a heads up which also occurred to me while reviewing https://github.com/ome/docker-example-omero/issues/13 is that we probably should have a general handle on accounts for whom we use docker images, either as a list or guidelines on yes&no.

Comment (jburel) [reply molecule/default/molecule.yml:12]: This is a simple one that we could easily handle ourselves if we think it is preferable

File: molecule/default/requirements.yml [hunk 2]
```diff
@@ -1,2 +1,7 @@
 ---
-- src: ome.nfs_mount
+- name: nfs_mount
+  src: https://github.com/pwalczysko/ansible-role-nfs-mount/
+  version: Rocky9
+- name: nfs_share
+  src: https://github.com/pwalczysko/ansible-role-nfs-share/
```
Comment (jburel) [inline molecule/default/requirements.yml]: why?

Comment (jburel) [reply molecule/default/requirements.yml]: could you point to ome

Comment (pwalczysko) [reply molecule/default/requirements.yml]: > could you point to ome

Done in https://github.com/ome/ansible-role-nfs-share/pull/7/commits/b59fb58e1f59325c103d2dc0892af5cc01fa3a3e, ta

**Comment by pwalczysko** [comment]
@jburel This is passing the test now, thank you

**Pull request merged**